backend/api/Helpers/Mapping/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,24 @@
+using Mapster;
+using MapsterMapper;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Reflection;
+
+namespace Pims.Api.Helpers.Mapping
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection AddMapster(this IServiceCollection services, Action<TypeAdapterConfig> options = null)
```

Comment:
Makes the `Startup` class code cleaner.  This allows us to use DI for the `IMapper` object.